cli/sawtooth_cli/rest_client.py

```diff
@@ -56,7 +56,11 @@ def send_batches(self, batch_list):
         Returns:
             dict: the json result data, as a dict
         """
-        data_bytes = batch_list.SerializeToString()
+        try:
+            data_bytes = batch_list.SerializeToString()
+        except AttributeError:
+            data_bytes = batch_list
+
```

Comment:
As the Rest Client is currently just an internal helper for CLI, tests and the like, I think a whatever-is-most-convenient implementation is fine. If we want to package the Rest Client for external usage, there is probably a fair amount of design and refactoring that should happen anyway.